### PR TITLE
Feature/update input field hint text

### PR DESCRIPTION
**Pull request by eskilsen**
## Description

Update input field group with hint texts. Tooltips is out, different hint texts are in. Mainly to be wcag complient. 
Also updated documentation with more examples and a more crisp look with tabs in componentpreview. 

## How Has This Been Tested?

Ran all existing tests and updated/removed tests. 
## Screenshots (if appropriate):
Before: 
image
Now:
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Review by PhungNg — CHANGES_REQUESTED** [review]
Some few changes before we can merge !
Otherwise..Great job ☜(ﾟヮﾟ☜)

File: src/App/ComponentsDocumentation/components/InputField/constants.js [hunk 1]
```diff
@@ -1,122 +1,136 @@
...
-export const textareaShowCase = {
-    id: "no-tabs-2",
-    elements: [
+
+            </>
+        },
         {
-            component: <InputGroup type="textarea" label="Input label" placeholder="Placeholder text" />,
+            tab: "Text area",
+            component: <InputGroup type="textarea" label="Label" />,
```
[the remark this answers is not in the document]
Comment (eskilsen) [reply src/App/ComponentsDocumentation/components/InputField/constants.js:109]: Hmm... The hint text is there to provide essential information for the user.. If something is disabled they should still be able to read the info.. Dont'cha agree? Maybe the info can help the users make the components enabled again? 🤓

File: src/App/ComponentsDocumentation/components/Select/constants.js [hunk 2]
```diff
@@ -15,17 +15,24 @@ export const selectOverview = {
...
+                                    helpBlock: "Hint text"
+                                }
+                            },
+                            {
+                                id: "expander_hint_text",
+                                name: "Expander hint",
+                                value: {
+                                    expandingHintText: "Expanding hint text",
                                 }
                             }
```
Comment (PhungNg) [inline src/App/ComponentsDocumentation/components/Select/constants.js:37]: As we discussed :)

image

**Review by PhungNg — CHANGES_REQUESTED** [review]
Cooool! Just one small thing :) 🎖️

**Review by PhungNg — APPROVED** [review]
Niceee! ⭐

**Pull request merged**